packages/master-detail-layout/src/vaadin-master-detail-layout.js

```diff
@@ -152,6 +152,15 @@ class MasterDetailLayout extends ResizeMixin(ElementMixin(ThemableMixin(PolylitM
       :host([has-detail-min-size][orientation='vertical']:not([overlay])) [part='detail'] {
         min-height: var(--_detail-min-size);
       }
+
+      /* Stack mode */
+      :host([stack]) [part='master'] {
+        max-height: 100%;
+      }
+
+      :host([stack]) [part='detail'] {
+        inset: 0;
```

Comment:
We also need to unset `width` or `height` here in case the details have a fixed size.

Reply:
Thanks, fixed by adding `:not([stack])` to corresponding selectors.